website/content/_index.md

```diff
@@ -16,22 +16,86 @@ Wash helps you deal with all your remote or cloud-native infrastructure using th
+EC2 instances             | `aws ec2 describe-instances --profile foo --query 'Reservations[].Instances[].InstanceId' --filters Name=tag-key,Values=owner --output text`
+Docker containers         | `docker ps --filter “label=owner”`
+Kubernetes pods           | `kubectl get pods --all-namespaces --selector=owner`
+Google Compute instance   | `gcloud compute instances list --filter=”labels.owner:*”`
+
+That’s a lot of commands you have to use, applications you need to install, and DSLs you have to learn just to do some very fundamental and basic tasks. Now take a look at how you’d perform those same tasks with Wash:
+
+<div style="width:200px">List all</div> | 
+----------------------|---
+AWS EC2 instances     | `find aws/foo -k '*ec2*instance'`
+Docker containers     | `find docker -k '*container' `
+Kubernetes pods       | `find kubernetes -k '*pod'`
+GCP Compute instances | `find gcp -k '*compute*instance'`
+
+<div style="width:200px">Read</div>         | 
+--------------------------------------------|---
+Console output of an EC2 instance           | `cat aws/foo/resources/ec2/instances/bar/console.out`
+Console output of a Google compute instance | `cat gcp/<project>/compute/foo/console.out`
+An S3 object's content                      | `cat aws/foo/resources/s3/bar/baz`
+A GCP Storage object's content              | `cat gcp/<project>/storage/foo/bar`
+
+<div style="width:200px">Exec `uname` on </div> | 
+-----------------------------|---
+An EC2 instance              | `wexec aws/foo/resources/ec2/instances/bar uname`
+An a Docker container        | `wexec docker/containers/foo uname`
+Exec on a Kubernetes pod     | `wexec kubernetes/<context>/<namespace>/pods/foo uname`
+On a Google Compute instance | `wexec gcp/<project>/compute/foo uname`
+
+<div style="width:200px">Find by 'owner' tag/label</div> | 
+------------------------|---
+EC2 instances           | `find aws/foo -k '*ec2*instance' -meta '.tags[?].key' owner`
+Docker containers       | `find docker -k '*container' -meta '.labels.owner' -exists`
+Kubernetes pods         | `find kubernetes -k '*pod' -meta '.metadata.labels.owner' -exists`
+Google Compute instance | `find gcp -k '*compute*instance' -meta '.labels.owner' -exists`
+
+From the table, we see that using Wash means:
+
+* You no longer have to learn different commands to execute a task across different things. All you need is one command (`find` for List/Find; `cat` for Read; and `wexec` for Exec).
+
+* You no longer have to install a bunch of different tools. All you need to install is the Wash binary.
+
+* You no longer have to learn different DSLs for filtering stuff. All you need to learn is find's expression syntax and its individual primaries. Once you do that, you can filter on almost any conceivable property of your specific thing.
+
+And this is only scratching the surface of Wash's capabilities. Checkout the screencast below to see some more (and to see Wash in action):
 
-Wash aims to:
```

Comment:
I feel like some of this is still useful and missing from the introduction above.

Reply:
Primarily about building better tools across a variety of cloud APIs and being easy to extend.

Reply:
When I tried to precisely define what we mean by "building better tools across a variety of cloud APIs," I ended up reducing the problem down to "centralizing a bunch of basic and fundamental stuff" because when you have that down, the "better tools" come naturally. The tables in the intro already hint at the latter, so I don't know if stating the "better tools" stuff is necessary. I also think that's something we should show people via a collage of interesting problems that are extremely easy to do with Wash (e.g. like navigating stuff, tailing a bunch of logs, getting SSHFS-like functionality for free on a bunch of different things, finding/filtering things, etc.). I'm thinking of adding some more of those to the screencast, but I also think a separate section of "Here are some cool, non-trivial sysadmin problems that are really easy to do with Wash" would be nice (we could tack those on as blog posts).

I think making a note of Wash being easy to extend is a good idea. Where do you think that should go?

Reply:
Theme is instead of telling people "You can build better stuff with Wash!", we should show them what building stuff with Wash would look like. The product's good enough that the "Oh my god, I really can build better tools with this" insight will likely come naturally to the user.

Reply:
Extending can probably be a note at the end.

Reply:
Per discussion on Slack, think it's better to leave "Extending Wash" to the features section. The intro contains enough information already that adding an additional "NOTE: You can extend Wash via ..." is not worth imposing a context-switch of "Ok, from centralizing commands => external plugins" on the reader.